src/components/noticeBanner.jsx

```diff
@@ -9,14 +9,15 @@ const NoticeBanner = () => {
         NOTICE
       </Text>
       <Text fontSize='md'>
-        The DAOhaus app isn't currently able to display the most recent onchain
-        data for any Gnosis Chain DAO. This is due to TheGraph subgraph errors.{' '}
+        The Rinkeby testnet is nearing end-of-life and our support is moving to
```

Comment:
Rinkeby testnet is nearing end-of-life and our support is moving to Goerli. Please summon new test DAOs on Goerli. DAOs on Rinkeby will not be supported after September 30, 2022.